EIPS/eip-1108.md

```diff
@@ -33,12 +33,48 @@ note](https://github.com/ethereum/go-ethereum/pull/16301#issuecomment-372687543)
+  [Matter Labs](https://github.com/matter-labs/matter-network) utilizes the precompiles to implement Ignis, a scaling solution with a throughput of 500txns per second  
+  [Rollup](https://github.com/rollup/rollup) utilizes the precompiles to create L2 scaling solutions, where the correctness of transactions is gauranteed by main-net, without an additional consensus layer  
+  [ZEther](https://crypto.stanford.edu/~buenz/papers/zether.pdf) uses precompiles `0x06` and `0x07` to construct confidential transactions  
+
+These are all technologies that have been, or are in the process of being, deployed to main-net. There protocols would all benefit from reducing the gas cost of the precompiles.  
+
+To give a concrete example, it currently costs `820,000` gas to validate the cryptography in a typical AZTEC confidential transaction. If the gas schedule for the precompiles correctly reflected their load on the Ethereum network, this cost would be `197,000` gas. This significantly increases the potential use cases for private assets on Ethereum. AZTEC has a number of cryptographic protocols that we plan deploy to Ethereum, but are at the limits of what is practical given the current precompile costs:  
+
+* Confidential weighted voting  
+* Partial-order filling over encrypted orders, for private decentralized exchanges  
+* Anonymous identity sharing proofs (e.g. proving you are on a whitelist, without revealing who you are)  
+* Many-to-one payments and one-to-many confidential payments, as encrypted communication channels between main-net and L2 applications  
+
+For ZK-SNARK based protocols on Ethereum, EIP-1108 will not only reduce the gas costs of verifying ZK-SNARKs substantially, but can also aid in [batching together multiple ZK-SNARK proofs](https://github.com/matter-labs/Groth16BatchVerifier). This is also a technique that can be used to split up monolithic ZK-SNARK circuits into a batch of ZK-SNARKs with smaller individual circuit sizes, which makes ZK-SNARKs both easier to construct and deploy.
+
+ZEther transactions currently cost ~`6,000,000` gas. This EIP would would reduce this to ~`1,000,000` gas, which makes the protocol more practical.  
+
+To summarise, the are a number of protocols that currently exist on main-net, that would benefit immensely from this EIP. Elliptic curve cryptography can provide valuable solutions for Ethereum, such as scaling and privacy, and the scope and scale of these solutions can be increased if the gas costs for the `bn128` precompiles accurately reflects their computational load on the network.
+
+### Cheaper elliptic curve cryptography can be used to trade storage for computation  
+
+Solutions such as Rollup and Ignis can be used to batch groups of individal transactions into a zk-snark proof, with the on-chain state being represented by a small Merkle root, instead of a large number of account balances.  
```

Comment:
```suggestion
Solutions such as Rollup and Ignis can be used to batch groups of individal transactions into a zk-SNARK proof, with the on-chain state being represented by a small Merkle root, instead of a large number of account balances.  
```